docs/src/charm/howto/install-terraform.md

```diff
@@ -0,0 +1,124 @@
+# Installing with Terraform
+
+This guide walks you through the process of installing {{ product }} using
+the [Terraform Juju Provider][juju-provider-tf]. 
+
+## Prerequisites
+
+This guide requires the following:
+
+- A Juju controller and model 
+<!-- TODO remove juju prerequisites once ground up module is available -->
+- The Terraform cli, which can be installed via the [snap store][terraform]
+
+## Authentication
+
+As a first step, authenticate the Terraform Juju Provider with the juju
+controller by choosing one of the options outlined in the
+[provider documentation][auth].
+
+## Terraform Module creation
+
+The Terraform deployment is done using a high-level module that specifies the
+juju model to deploy the submodules to. The k8s charm and the k8s-worker charm
+each live in a separate module referenced by the high-level module.
```

Comment:
Also a small change:
The high-level module also references the k8s charm and the k8s-worker charm that each live in a separate module.